Modules/options.c

```diff
@@ -261,6 +262,12 @@ LDAP_get_option(LDAPObject *self, int option)
     Py_ssize_t i, num_extensions;
 
     switch (option) {
+#ifdef HAVE_SASL
+    case LDAP_OPT_X_SASL_SSF_EXTERNAL:
+        /* Read-only options */
+        PyErr_SetString(PyExc_ValueError, "write-only option");
```

Comment:
Could you also test that this error is raised?

Reply:
Done